docs/source/kinds_of_types.rst

```diff
@@ -566,6 +566,26 @@ Python 3.6 introduced an alternative, class-based syntax for named tuples with t
 
     p = Point(x=1, y='x')  # Argument has incompatible type "str"; expected "int"
 
+.. note::
+
+  You can use raw ``NamedTuple`` pseudo-class to annotate type
```

Comment:
Can you explicitly document that this is an experimental, non-standard extension that may not be supported by other Python tools?